### DISCUSSION TITLE
Minimal setup for cloning my own voice in Finnish

### DISCUSSION THREAD

**Discussion by timolaine** [coqui-ai/TTS#3040] [General Q&A]
My goal is to clone my own voice with Coqui-AI in Finnish with minimal hassle.

I have spent a few hours of looking at Coqui-AI. I managed to get it installed on my Mac and I also managed to get it to synthesize speech with a command like this:

`tts --text "Tämä on suomenkielistä tekstiä." --model_name "tts_models/fi/css10/vits" --out_path output.wav`

I am not a Python or AI guru, and would not like to get too deep into configuring small details. I am satisfied with average quality, at least in the beginning. I have made a few samples of my own voice and written their metadata in the style of:

```
audio1||Tämä on se mitä puhun.
audio2||Myös tämä on puhettani.
```

However, I am struggling to understand what I need to do next, so I have been blindly poking around and running different commands with different parameters. What would be the simplest configuration and commands to get started with this?

**Comment by audiocommander** [comment]
Hey, not sure if this helps, as there should be a couple of options from scripts to cluster computing. The TTS directory has a subfolder "notebooks", which contains really nice and well documented Jupyter Notebooks. I'm on a Mac and installed Jupyter Lab via homebrew, pretty easy. With Jupyter Lab you can then open the TTS notebooks in your browser and execute the code inside.

**Reply by audiocommander** [reply]
PS: also see https://tts.readthedocs.io/en/latest/index.html
